_posts/2016-12-30-foreman-community-newsletter---december-2016.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Foreman Community Newsletter - December 2016
+date: 2016-12-28 12:42:23
+author: Ondřej Pražák
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Foreman 1.14, Katello 3.3, plugin updates, YouTube content, new releases...
+  in the very last Foreman blog post of 2016.
+---
+
+### Foreman 1.14 branched
```

Comment:
There have been 2 people complaining about wrong signatures on IRC - test it on your own and add a comment if it doesn't work. centos-release-scl has a different signature to other packages in there